### PR TITLE
ruff compliance for D205.

### PR DESCRIPTION
**Pull request by tkknight**
### Description
PR solely for fixing `D205: 1 blank line required between summary line and description`

Continues work of https://github.com/SciTools/iris/issues/5625.

---
[Consult Iris pull request check list]( https://scitools-iris.readthedocs.io/en/latest/developers_guide/contributing_pull_request_checklist.html)

### CODE REVIEW

File: lib/iris/fileformats/cf.py [hunk 1]
```diff
@@ -373,7 +383,9 @@ def identify(cls, variables, ignore=None, target=None, warn=True):
         return result
 
     def spans(self, cf_variable):
-        """Determine whether the dimensionality of this variable
+        """Determine the dimensionality of this variable is a subset.
```
Comment (bjlittle) [inline lib/iris/fileformats/cf.py:386]:
```suggestion
        """Determine dimensionality coverage.
```

File: lib/iris/fileformats/cf.py [hunk 2]
```diff
@@ -445,7 +459,9 @@ def identify(cls, variables, ignore=None, target=None, warn=True):
         return result
 
     def spans(self, cf_variable):
-        """Determine whether the dimensionality of this variable
+        """Determine whether the dimensionality of this variable is a subset.
```
Comment (bjlittle) [inline lib/iris/fileformats/cf.py:462]:
```suggestion
        """Determine dimensionality coverage.
```

File: lib/iris/fileformats/cf.py [hunk 3]
```diff
@@ -777,7 +801,9 @@ def cf_label_dimensions(self, cf_data_var):
         )
 
     def spans(self, cf_variable):
-        """Determine whether the dimensionality of this variable
+        """Determine if dimensionality is a subset of the specified target variable.
```
Comment (bjlittle) [inline lib/iris/fileformats/cf.py:804]:
```suggestion
        """Determine dimensionality coverage.
```

File: lib/iris/fileformats/_structured_array_identification.py [hunk 4]
```diff
@@ -92,7 +100,9 @@ def __new__(cls, stride, unique_ordered_values):
 
     @property
     def size(self):
-        """The ``size`` attribute is the number of the unique values in the
+        """The ``size`` attribute is the number of the unique values in the original array.
```
Comment (bjlittle) [inline lib/iris/fileformats/_structured_array_identification.py:103]:
```suggestion
        """Number of unique values in the original array.
```

File: lib/iris/fileformats/_structured_array_identification.py [hunk 5]
```diff
@@ -114,7 +124,9 @@ def __ne__(self, other):
         return not (self == other)
 
     def construct_array(self, size):
-        """The inverse operation of :func:`ArrayStructure.from_array`, returning
+        """The inverse operation of :func:`ArrayStructure.from_array`.
```
Comment (bjlittle) [inline lib/iris/fileformats/_structured_array_identification.py:127]:
```suggestion
        """Build 1D array.
```

File: lib/iris/coords.py [hunk 6]
```diff
@@ -802,7 +783,9 @@ def shape(self):
         return self._values_dm.shape
 
     def xml_element(self, doc):
-        """Create the :class:`xml.dom.minidom.Element` that describes this
+        """Create :class:`xml.dom.minidom.Element` that describes :class:`_DimensionalMetadata`.
```
Comment (bjlittle) [inline lib/iris/coords.py:786]:
```suggestion
        """Create XML element.
```

File: lib/iris/coords.py [hunk 7]
```diff
@@ -1212,7 +1191,9 @@ def _get_2d_coord_bound_grid(bounds):
 
 
 class Cell(namedtuple("Cell", ["point", "bound"])):
-    """An immutable representation of a single cell of a coordinate, including the
+    """An immutable representation of a single cell of a coordinate.
```
Comment (bjlittle) [inline lib/iris/coords.py:1194]:
```suggestion
    """An coordinate cell containing a single point, or point and bounds.
```

File: lib/iris/coords.py [hunk 8]
```diff
@@ -1326,7 +1304,9 @@ def __ne__(self, other):
         return result
 
     def __common_cmp__(self, other, operator_method):
-        """Common method called by the rich comparison operators. The method of
+        """Common method called by the rich comparison operators.
```
Comment (bjlittle) [inline lib/iris/coords.py:1307]:
```suggestion
        """Common equality comparison.
```

File: lib/iris/coords.py [hunk 9]
```diff
@@ -1600,7 +1579,9 @@ def points(self, points):
 
     @property
     def bounds(self):
-        """The coordinate bounds values, as a NumPy array,
+        """The coordinate bounds values, as a NumPy array.
```
Comment (bjlittle) [inline lib/iris/coords.py:1582]:
```suggestion
        """Coordinate bounds values.
```

File: lib/iris/coords.py [hunk 10]
```diff
@@ -1639,7 +1620,9 @@ def coord_system(self, value):
 
     @property
     def climatological(self):
-        """A boolean that controls whether the coordinate is a climatological
+        """A boolean that controls whether the coordinate is a climatological time axis.
```
Comment (bjlittle) [inline lib/iris/coords.py:1623]:
```suggestion
        """Flag for representing a climatological time axis.
```

File: lib/iris/coords.py [hunk 11]
```diff
@@ -1748,14 +1724,13 @@ def core_bounds(self):
...
-        """Return a boolean indicating whether the coord's points array is a
-        lazy dask array or not.
-
-        """
+        """Return a boolean whether the coord's points array is a lazy dask array or not."""
         return super()._has_lazy_values()
 
     def has_lazy_bounds(self):
-        """Return a boolean indicating whether the coord's bounds array is a
+        """Return a boolean whether the coord's bounds array is a lazy dask array or not.
```
Comment (bjlittle) [inline lib/iris/coords.py:1731]:
```suggestion
        """Whether coordinate bounds are lazy.
```

File: lib/iris/coords.py [hunk 12]
```diff
@@ -1941,7 +1915,9 @@ def mod360_adjust(compare_axis):
         return contiguous, diffs
 
     def is_contiguous(self, rtol=1e-05, atol=1e-08):
-        """Return True if, and only if, this Coord is bounded with contiguous
+        """Check if the Coord is bounded with contiguous bounds.
```
Comment (bjlittle) [inline lib/iris/coords.py:1918]:
```suggestion
        """Whether coordinate has contiguous bounds.
```

File: lib/iris/coords.py [hunk 13]
```diff
@@ -1971,7 +1947,9 @@ def is_contiguous(self, rtol=1e-05, atol=1e-08):
         return contiguous
 
     def contiguous_bounds(self):
-        """Returns the N+1 bound values for a contiguous bounded 1D coordinate
+        """Return the N+1 bound values for a contiguous bounded 1D coordinate.
```
Comment (bjlittle) [inline lib/iris/coords.py:1950]:
```suggestion
        """Contiguous bounds of 1D coordinate.
```

File: lib/iris/coords.py [hunk 14]
```diff
@@ -2065,7 +2043,9 @@ def is_compatible(self, other, ignore=None):
 
     @property
     def bounds_dtype(self):
-        """The NumPy dtype of the coord's bounds. Will be `None` if the coord
+        """The NumPy dtype of the coord's bounds.
```
Comment (bjlittle) [inline lib/iris/coords.py:2046]:
```suggestion
        """The NumPy dtype of the coordinates bounds.
```

File: lib/iris/coords.py [hunk 15]
```diff
@@ -2087,8 +2067,11 @@ def has_bounds(self):
         return self._bounds_dm is not None
 
     def cell(self, index):
-        """Return the single :class:`Cell` instance which results from slicing the
+        """Return the single :class:`Cell` instance from slicing the points/bounds.
```
Comment (bjlittle) [inline lib/iris/coords.py:2070]:
```suggestion
        """The point/bound cell at the given coordinate index.
```

File: lib/iris/experimental/ugrid/mesh.py [hunk 16]
```diff
@@ -84,7 +84,9 @@
 
 
 class Connectivity(_DimensionalMetadata):
-    """A CF-UGRID topology connectivity, describing the topological relationship
+    """A CF-UGRID topology connectivity, describes the topological relationship.
```
Comment (bjlittle) [inline lib/iris/experimental/ugrid/mesh.py:87]:
```suggestion
    """CF-UGRID topology.
```

**Review by bjlittle — CHANGES_REQUESTED** [review]
@tkknight Gosh. This was hard.

You've make an incredible effort to reformat the docstrings. 

I've tried my best to complement your efforts to formulate an appropriate single line summary for each docstring. We don't need perfection here, as we can iterate over time and massage the content.

A bunch of suggestions which you can accept or not.

Either way, a huge effort and a decent step forwards :100: :+1:

**Comment by tkknight** [comment]
All review comments accepted - one at a time until I reliased there was a batch option!

One additional commit from my final read thru, minor layout and spotted some duplicate text.  See https://github.com/SciTools/iris/pull/5681/commits/d168d07a660affbf696e48876e24623fd8a516e8.

Think this is now ready.

**Review by bjlittle — APPROVED** [review]

**Pull request merged**